### PR TITLE
STU-2158-Geometry-task-improvements

### PR DESCRIPTION
**Pull request by filip-povolny-studyly**
* add type definitions
* add build directory

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "tex2max",
   "version": "1.3.1",
   "description": "LaTeX math to Maxima code converter",
-  "main": "lib/tex2max.common.js",
+  "main": "build/merge.js",
```
Comment (franz-schmidt-studyly) [inline package.json]: was this change intended?

Comment (filip-povolny-studyly) [reply package.json]: Yes, it's needed for correct `tex2max` import in webapp. `lib` directory does not even exist in the repo and is probably an output directory of the `build` script.

Comment (filip-povolny-studyly) [reply package.json]: So I managed to build tex2max and it works with `lib/tex2max.common.js` as well. Should I add the lib folder to git?

Comment (franz-schmidt-studyly) [reply package.json]: what do you mean it works with `lib/tex2max.common.js` is this a dependency? if it is inlcuded via a dependency you don't need to add it otherwise what would happen if we don't add the lib folder? is building even possible without it?

Comment (filip-povolny-studyly) [reply package.json]: No, lib not a dependency. It is a directory created by building tex2max. Similar to mathquill, ckeditor5 and probably all the custom node modules, we track the build directories with version control. `lib/tex2max.common.js` is the built tex2max library that should be used by the webapp.

Comment (franz-schmidt-studyly) [reply package.json]: so webapp does not work without the lib folder and building is not part of the deploy?

Comment (filip-povolny-studyly) [reply package.json]: * webapp does not work without the lib folder: it does, but then tex2max must be imported from `build/merge.js`
* building is not part of the deploy: true, it is not

Comment (franz-schmidt-studyly) [reply package.json]: I see, I guess than this totally fine

**Review by franz-schmidt-studyly — APPROVED** [review]
